app/javascript/hyacinth_v1/components/digital_objects/DigitalObjectInterface.js

```diff
@@ -1,22 +1,53 @@
 import React from 'react';
 import { capitalize } from 'lodash';
 import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
 
 import Tab from '../shared/tabs/Tab';
 import Tabs from '../shared/tabs/Tabs';
 import TabBody from '../shared/tabs/TabBody';
 import ContextualNavbar from '../shared/ContextualNavbar';
+import { Button } from 'react-bootstrap';
 import DigitalObjectSummary from './DigitalObjectSummary';
+import { redirectToSearch } from '../../utils/redirectToSearch';
 
 function DigitalObjectInterface(props) {
   const { digitalObject, children } = props;
   const { id, title, digitalObjectType } = digitalObject;
+  const history = useHistory();
+  const latestSearchQueryString = sessionStorage.getItem('searchQueryParams');
+
+
+  const backToSearchHandler = () => {
+    const latestSearchQuery = JSON.parse(latestSearchQueryString);
+    redirectToSearch(history, latestSearchQuery);
+  };
+
+  const backToSearchButton = () => {
+    const button = latestSearchQueryString ?
```

Comment:
I'd recommend directly returning the output of the ternary operator rather than assigning to local variable `button`.